src/views/SessionView/index.vue

```diff
@@ -140,6 +146,9 @@ export default {
   },
   sockets: {
     bump: function() {
+      window.alert(
+        "Another volunteer is already helping this student. We apologize for the inconvenience. Thank you for volunteering with UPchieve!"
```

Comment:
Could we display a different message if the session has already ended when a volunteer joins?